pravega/src/main/scala/akka/stream/alpakka/pravega/PravegaSettings.scala

```diff
@@ -365,22 +377,31 @@ object TableReaderSettingsBuilder {
    * Create reader settings from the default configuration
    * `akka.alpakka.pravega`.
    */
-  def apply[K, V](actorSystem: ActorSystem): TableReaderSettingsBuilder[K, V] =
-    apply(actorSystem.settings.config.getConfig(configPath))
+  def apply[K, V](implicit actorSystem: ActorSystem,
+                  keySerializer: Serializer[K],
+                  valueSerializer: Serializer[V]): TableReaderSettingsBuilder[K, V] =
+    apply(actorSystem.settings.config.getConfig(configPath), keySerializer, valueSerializer)
 
   /**
    * Create reader settings from the default configuration
    * `akka.alpakka.pravega`.
    */
-  def create[K, V](actorSystem: ActorSystem): TableReaderSettingsBuilder[K, V] =
-    apply(actorSystem)
+  def create[K, V](implicit actorSystem: ActorSystem,
```

Comment:
```suggestion
  def create[K, V](actorSystem: ActorSystem,
```
This is for the Java API which shouldn't use `implicit`.